File: src/main/java/com/grepp/curdsample/util/PriorityResolver.java
package com.grepp.curdsample.util;

public class PriorityResolver {

    public static String resolve(Integer priority) {
        return switch (priority) {
            case 0 -> " ";
            case 1 -> "text-red-700";
            case 2 -> "text-red-500";
            case 3 -> "text-yellow-500";
            case 4 -> "text-green-500";
            default -> "text-green-700";
        };
    }

}

File: src/main/java/com/est/curdsample/util/PriorityResolver.kt
package com.est.curdsample.util

fun priorityResolve(priority: Int): String {
    return when (priority) {
        0 -> " "
        1 -> "text-red-700"
        2 -> "text-red-500"
        3 -> "text-yellow-500"
        4 -> "text-green-500"
        else -> "text-green-700"
    }
}
